CMakeLists.txt

```diff
@@ -138,6 +138,16 @@ else()
 	set(LIBSERIALPORT_INCLUDE_DIR "")
 endif()
 
+if ((NOT LIBAD9166_LIBRARIES) AND (NOT LIBAD9166_INCLUDE_DIRS))
+	set(LIBAD9166_LIBRARIES "")
+	set(LIBAD9166_INCLUDE_DIRS "")
+endif()
+
+if ((NOT LIBAD9361_LIBRARIES) AND (NOT LIBAD9361_INCLUDE_DIRS))
+	set(LIBAD9361_LIBRARIES "")
+	set(LIBAD9361_INCLUDE_DIRS "")
+endif()
```

Comment:
Do we need to do this extra step? Can't we do `if (NOT LIBAD9361_LIBRARIES)` and print the message plus the removal?

Reply:
If we don't do:
```
set(LIBAD9166_LIBRARIES "")
set(LIBAD9166_INCLUDE_DIRS "")
```
the error will persist:
`CMake Error: The following variables are used in this project, but they are set to NOTFOUND.`

with the current structure of the CMakeLists.txt

I wanted to keep the same approach as for libserialport:
https://github.com/analogdevicesinc/iio-oscilloscope/blob/73a5a40a4c720c88650ef0dad7a77214a5d493e8/CMakeLists.txt#L130-L139

Reply:
Ahh I see this is needed because of `set(LIBOSC_LIBS...)`